server/mux.go

```diff
@@ -313,8 +313,13 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 
 	// Global application config for Chronograf
 	router.GET("/chronograf/v1/config", EnsureSuperAdmin(service.Config))
-	router.GET("/chronograf/v1/config/:section", EnsureSuperAdmin(service.ConfigSection))
-	router.PUT("/chronograf/v1/config/:section", EnsureSuperAdmin(service.ReplaceConfigSection))
+	router.GET("/chronograf/v1/config/auth", EnsureSuperAdmin(service.AuthConfig))
+	router.PUT("/chronograf/v1/config/auth", EnsureSuperAdmin(service.ReplaceAuthConfig))
+
+	// Organization config settings for Chronograf
+	router.GET("/chronograf/v1/org_config", EnsureViewer(service.OrganizationConfig))
```

Comment:
Does it make sense that a viewer can see these values, but not set them?

Reply:
`Config` only contains `Auth` and `Org_Config` only contains `LogViewer`, both of which can be updated via other routes, but the user can't update the entire `Config` or `Org_Config` objects

Reply:
ah makes sense